src/main/resources/swagger.api/schemas/publication_schema.yaml

```diff
@@ -25,13 +25,47 @@ PublicationResponse:
   properties:
     id:
       description: id of publication record
-      type: string
+      $ref: "common.yaml#/uuid"
     status:
       type: string
       $ref: "publication_schema.yaml#/PublicationStatus"
+  additionalProperties: false
+
 
+PublicationDetailsResponse:
+  type: object
+  title: Publication details response
+  properties:
+    id:
+      description: id of publication record
+      $ref: "common.yaml#/uuid"
+    status:
+      type: string
+      $ref: "publication_schema.yaml#/PublicationStatus"
+    dateTime:
+      description: the date of publication was created
+      type: string
+    request:
+      description: tenant request payload
+      type: string
+    errors:
+      description: "List of errors"
+      type: array
+      items:
+        type: object
+        $ref: "publication_schema.yaml#/PublicationStatusError"
   additionalProperties: false
 
 PublicationStatus:
   description: publication status
   enum: [ "IN_PROGRESS", "ERROR", "COMPLETE" ]
+
+PublicationStatusError:
+  description: publication status error
+  properties:
+    tenantId:
```

Comment:
I propose to add also status code here in response, especially to distinguish 400x and 500x errors, this field already exists in db table

Reply:
added errorCode